frontend/src/scenes/session-recordings/player/inspector/playerInspectorLogic.ts

```diff
@@ -2,6 +2,7 @@ import { customEvent, eventWithTime } from '@rrweb/types'
 import FuseClass from 'fuse.js'
 import { actions, connect, events, kea, key, listeners, path, props, propsChanged, reducers, selectors } from 'kea'
 import { loaders } from 'kea-loaders'
+// import { subscriptions } from 'kea-subscriptions'
```

Comment:
```suggestion
```

Reply:
Thanks, I'd forgotten to push my last commit